assignment-client/src/entities/EntityTreeSendThread.cpp

```diff
@@ -118,13 +120,9 @@ void EntityTreeSendThread::traverseTreeAndSendContents(SharedNodePointer node, O
                             if (_traversal.getCurrentView().cubeIntersectsKeyhole(cube)) {
                                 float priority = _conicalView.computePriority(cube);
                                 if (priority != PrioritizedEntity::DO_NOT_SEND) {
-                                    float renderAccuracy = calculateRenderAccuracy(_traversal.getCurrentView().getPosition(),
-                                                                                   cube,
-                                                                                   _traversal.getCurrentRootSizeScale(),
-                                                                                   _traversal.getCurrentLODOffset());
-
-                                    // Only send entities if they are large enough to see
-                                    if (renderAccuracy > 0.0f) {
+                                    float distance = (glm::distance(cube.calcCenter(), viewPosition) + MIN_VISIBLE_DISTANCE);
```

Comment:
Extra parens, and we should maybe consider putting this calculation in a function so if it changes we don't have to modify it everywhere.